tests/test_wave_03.py

```diff
@@ -119,31 +119,23 @@ def test_mark_incomplete_on_incomplete_task(client, one_task):
     assert Task.query.get(1).completed_at == None
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
+# @pytest.mark.skip(reason="No way to test this feature yet")
 def test_mark_complete_missing_task(client):
     # Act
     response = client.patch("/tasks/1/mark_complete")
     response_body = response.get_json()
 
     # Assert
     assert response.status_code == 404
+    assert response_body == {"message": "Task 1 not found"}
```

Comment:
Nice work with this test completion! This assertion is a great way to validate functionality is working as expected